pulsar-functions/instance/src/main/python/python_instance.py

```diff
@@ -26,7 +26,10 @@
 import os
 import signal
 import time
-import Queue
+try:
```

Comment:
it might be better to check python version, rather than try except?

Reply:
try/except is a fairly common practice in my experience.